video/image_writer.c

```diff
@@ -73,6 +80,8 @@ const struct m_option image_writer_opts[] = {
     {"webp-lossless", OPT_FLAG(webp_lossless)},
     {"webp-quality", OPT_INT(webp_quality), M_RANGE(0, 100)},
     {"webp-compression", OPT_INT(webp_compression), M_RANGE(0, 6)},
+    {"jxl-distance", OPT_DOUBLE(jxl_distance), M_RANGE(0.0, 15.0)},
+    {"jxl-effort", OPT_INT(jxl_effort), M_RANGE(1, 9)},
```

Comment:
Do we have a policy on whether config options should be exposed for features that aren't present? I think we've got examples of both today, but removing them when the feature is missing makes it clearer that the feature is not there. @jeeb ?